testutils/src/main/java/com/amplifyframework/testutils/Latch.java

```diff
@@ -27,7 +27,7 @@
  */
 @SuppressWarnings({"SameParameterValue", "WeakerAccess"})
 final class Latch {
-    private static final long REASONABLE_WAIT_TIME_MS = TimeUnit.SECONDS.toMillis(5);
+    static final long REASONABLE_WAIT_TIME_MS = TimeUnit.SECONDS.toMillis(5);
```

Comment:
can this remain `private`? Who else needs to know about this value?

Reply:
This was for my hub accumulator change, but I'll revert this change along-side the hub accumulator